### PR TITLE
Implement @-symbols in chat

### PR DESCRIPTION
**Pull request by JacksonKearl**
This is a rough initial implementation. 

Some notes: 
- there doesn't seem to be a way to incrementally update the tree sitter parser's project. As such this currently reparses much more than needed.
- the file icon is currently reused for symbols, does anyone know how the `icons/*.svelte` files were sourced?

### CODE REVIEW

**Comment by jesse-michael-han** [comment]
Great, let's put incremental `IR.Project` updates on the task queue and use some sort of `</>` icon for code objects.

File: editors/rift-vscode/src/util/AtableFileFunction.ts [hunk 1]
```diff
@@ -3,10 +3,11 @@ import { AtableFile } from "../types";
 export const AtableFileFromUri = (Uri: vscode.Uri): AtableFile => {
   return {
     fileName: Uri.path.split("/").pop() ?? Uri.path,
-    fullPath: Uri.fsPath,
+    fullPath: Uri.fsPath + (Uri.fragment? '#' + Uri.fragment : ''),
     fromWorkspacePath: vscode.workspace.asRelativePath(Uri),
```
Comment (cristianoc) [inline editors/rift-vscode/src/util/AtableFileFunction.ts:7]: this gives a relative path

Comment (JacksonKearl) [reply editors/rift-vscode/src/util/AtableFileFunction.ts:7]: Could you elaborate? VSCode's URI's fsPath is a full path suitable for interacting with system file utilities.

Comment (cristianoc) [reply editors/rift-vscode/src/util/AtableFileFunction.ts:7]: Forgot to remove this comment.
Ignore.

**Pull request merged**